### ISSUE TITLE
Main Page Styling

### ISSUE THREAD

**Issue by NickSramcik** [NickSramcik/banki-brunch#6]
We now have a functional basic layout, but it could use some extra flair. Perhaps some breakfast/brunch theming? I'm open to suggestions. We should consider having a consistent color palette as well.

**Comment by funbunch** [comment]
I agree! Daisy has quite a few themes. And also, I wasn't sure if we want more of a proper logo styled. Open too!

**Comment by DevKnightRuin** [comment]
![image](https://github.com/NickSramcik/banki-brunch/assets/29897228/d17f6938-fe46-4f75-8ff9-3c36f41db23d)

**Comment by Angela-LJ** [comment]
Breakfast themed is a cool idea! Perhaps we can think of a way to blend the themes of programming with breakfast?

**Comment by DevKnightRuin** [comment]
Probably best for a neutral colors for boarders and such, but keep the images with breakfast items (bacon, eggs, waffles etc).  I just realized the palette I posted didn't have a nice looking yellow XD

**Comment by Angela-LJ** [comment]
When I think of breakfast I think of the color orange (probably thinking of orange juice lol). Should we have orange in the palette?

**Comment by DevKnightRuin** [comment]
If we do neutral colors for the borders, we could get random breakfast images each time we pull a question.  Maybe not full images of breakfast plates, but like different breakfast items thrown around like bacon, eggs, various fruits, etc.  Just a thought.  I do like that image with the toast and fruits.  Good thing I already ate some before seeing that ^_^

**Comment by Angela-LJ** [comment]
I kind of want to try my hand at styling this if no one else wants it. I think I can make something creative out of it, and my buddy Jordan (Gang/Jdub679) wants to join me with styling it as well! We can make some mock ups on Figma first before we commit to a particular design.

**Comment by Jdub679** [comment]
Me and Angela can work on this and make a few designs. I have experience with react/jsx, and we've used tailwind as well

**Comment by NickSramcik** [comment]
I really like those color palettes cblanked posted. I think something like the first palette with a shade of red added would be perfect.

**Comment by AhmedHHamdy** [comment]
@Angela-LJ @Jdub679 Hey guys, I would like to join you in styling the app. Is there a place available?

**Comment by cblanken** [comment]
We should definitely be using semantic naming for all our colors in Tailwind.
Here's an [article](https://dev.to/ynab/a-semantic-color-system-the-theory-hk7) with an example how you might configure your your colors.

But the general idea is to apply prefixes to your colors that have meaning for their usage not just the color they represent. For example you might choose `header-background`, `body-background`, `primary-accent`, `primary-text`. These all have particular usage in mind instead of color names like `orange-1`, `orange-2`, `dark-blue` etc.

This makes altering the color palette later if the design of the app changes or if we want to add distinct themes.

**Comment by intelagense** [comment]
Here is a view link for the weird thing I put together in Canva. I recolored it with colors from the color palettes. 
I do think the image is very "busy" with the horizontal lines from the pancake stack and the text overlay though.

![image](https://github.com/NickSramcik/banki-brunch/assets/18331781/899afc7d-b003-4afe-8af3-aa76d7883bde)

https://www.canva.com/design/DAFo0kpfrOs/JhUF0lHAMn6EXsXoQgrSgg/view?utm_content=DAFo0kpfrOs&utm_campaign=designshare&utm_medium=link&utm_source=publishsharelink

**Comment by intelagense** [comment]
![image](https://github.com/NickSramcik/banki-brunch/assets/18331781/662e22fc-689f-48bb-831e-90a217f282bd)

Just found this tool that lets you check all the contrast ratios: [Accessible Contrast Grid](https://contrast-grid.eightshapes.com/?version=1.1.0&background-colors=&foreground-colors=%23ffffff%0D%0A%23f5ad25%2C%20Gold%0D%0A%23fbd689%0D%0A%23f9e7cd%0D%0A%23a0774e%0D%0A%232f1707%0D%0A%23d36d6e%0D%0A%23b0302d%0D%0A&es-color-form__tile-size=compact&es-color-form__show-contrast=aaa&es-color-form__show-contrast=aa&es-color-form__show-contrast=aa18&es-color-form__show-contrast=dnp)
